### PR TITLE
Fix varPPO strategy and TSI indicator

### PR DESCRIPTION
**Pull request by Paulovms**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Bug fix.

* **What is the current behavior?** (You can also link to an open issue here)
TSI values on varPPO and TSI strategies stay above 95 after a lot of candles.
The first momentum value is the full close value and this put a big error on next EMAs.
varPPO use the PPO value from this.indicators.ppo.ppo to calculate PPO histogram, but this value is not been updated and are always 0. In fact, since Gekko 0.5.11, the PPO indicator exposes the PPO histogram directly to the API on this.indicators.ppo.result.ppohist.

* **What is the new behavior (if this is a feature change)?**
TSI indicator calculate the first value after get the first previous close and varPPO get ppoHist directly from PPO indicator.

### CODE REVIEW

File: exchanges/bitfinex.js [hunk 1]
```diff
@@ -39,7 +39,7 @@ var retryForever = {
 };
 
 // Probably we need to update these string
-var recoverableErrors = new RegExp(/(SOCKETTIMEDOUT|TIMEDOUT|CONNRESET|CONNREFUSED|NOTFOUND|StatusCodeError: 429|StatusCodeError: 5)/)
+var recoverableErrors = new RegExp(/(SOCKETTIMEDOUT|TIMEDOUT|CONNRESET|CONNREFUSED|NOTFOUND|429|443|5\d\d)/g);
```
Comment (askmike) [inline exchanges/bitfinex.js:42]: Is this supposed to be part of this PR?

**Review by askmike — COMMENTED** [review]
see question, will merge after you can let us know :)

File: exchanges/bitfinex.js (hunk 1 above)
Comment (Paulovms) [reply exchanges/bitfinex.js:42]: I’m sorry, this will update the Bitfinex string. Maybe I’ll submit another PR.

Comment (askmike) [reply exchanges/bitfinex.js:42]: Maybe it needs to be updated, did you add this specifically because it caught errors better?

Comment (Paulovms) [reply exchanges/bitfinex.js:42]: Yes, this string caught erros better.It can be committed.

**Comment by askmike** [comment]
thanks!

**Pull request merged**